core/server/src/main/java/alluxio/worker/netty/BlockDataServerHandler.java

```diff
@@ -80,14 +79,10 @@ void handleBlockReadRequest(final ChannelHandlerContext ctx, final RPCBlockReadR
     final long lockId = req.getLockId();
     final long sessionId = req.getSessionId();
 
-    BlockReader reader;
+    BlockReader reader = null;
     DataBuffer buffer;
     try {
       reader = mWorker.readBlockRemote(sessionId, blockId, lockId);
```

Comment:
@calvinjia / @gpang can you take a look at my change to exception handling here? Previously we would swallow any exception thrown by `readBlockRemote`. This doesn't seem right so I changed it, but want to make sure it wasn't intentional